src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -911,6 +911,8 @@ Public Class BuildDevDivInsertionFiles
                           <version>0.0</version>
                       </metadata>
                       <files>
+                          <file src="System.Collections.Immutable.dll"/>
```

Comment:
I thought this would be more readable and robust. In this case I don't necessarily want dedup, right? If we update the compiler files to include these binaries via listing in the CompilerFiles `List<T>` we don't want the concat to silently continue adding these binaries -- we want to get an error saying we've included the same thing in two places.